SampleProjects/TestSomething/excludeThis/exclude-this.cpp

```diff
@@ -0,0 +1,10 @@
+#include "test-something.h"
+int testSomething(void) {
+  millis();  // this line is only here to test that we're able to refer to the builtins
+  return 4;
+};
+
+int* aNullPointer(void) {
+  int* ret = nullptr;
+  return ret;
+}
```

Comment:
The contents of this directory appear to be a simple copy of files that already exist elsewhere in this project.

If the purpose of this file (and the `.h`) is just to cause some form of compilation error, I'd prefer that its contents speak entirely to that.  Just fill it with a syntax error in the form of 

```c++
This file was placed here specifically to break unit test compilation.  If arduino_ci is working properly, it should exclude this file (as per the .arduino-ci.yml configuration) and compilation should succeed. 
```

And name it something like `break-compilation.cpp`.